### PR TITLE
Add installers for linux arm64

### PR DESCRIPTION
**Pull request by ThadHouse**

### CODE REVIEW

File: WPILibInstaller-Avalonia/ViewModels/StartPageViewModel.cs [hunk 1]
```diff
@@ -171,10 +171,14 @@ public bool MissingResourceFiles
 
         private async Task<bool> SelectResourceFilesWithFile(string file)
         {
+            Console.WriteLine("Initial");
```
Comment (PeterJohnson) [inline WPILibInstaller-Avalonia/ViewModels/StartPageViewModel.cs:174]: I'm assuming these WriteLine's are for debugging purposes?

Comment (ThadHouse) [reply WPILibInstaller-Avalonia/ViewModels/StartPageViewModel.cs:174]: Yes. I'll remove them after testing.

**Review by PeterJohnson — APPROVED** [review]

**Comment by sciencewhiz** [comment]
Readme should be updated with the new architectures

**Review by auscompgeek — APPROVED** [review]
Fresh install tested on Linux x86-64.

**Pull request merged**